deltachat-ios/Controller/AccountSetupController.swift

```diff
@@ -284,14 +284,31 @@ class AccountSetupController: UITableViewController, ProgressAlertHandler {
     }()
 
     lazy var mvboxMoveCell: SwitchCell = {
-        return SwitchCell(
+        let cell = SwitchCell(
             textLabel: String.localized("pref_auto_folder_moves"),
             on: dcContext.getConfigBool("mvbox_move"),
             action: { cell in
+                self.dcAccounts.stopIo()
                 self.dcContext.setConfigBool("mvbox_move", cell.isOn)
+                self.dcAccounts.startIo()
+        })
+        cell.uiSwitch.isEnabled = !dcContext.getConfigBool("only_fetch_mvbox")
+        return cell
+    }()
+
+    lazy var watchMvboxCell: SwitchCell = {
```

Comment:
i'd name the cell as the flag it switches, so `onlyFetchMvboxCell`.

the name `watchMvboxCell` for `only_fetch_mvbox` is misleading.